### PR TITLE
CollaborationRequest

### PR DESCRIPTION
**Pull request by Arfaoui-khouloud**

### CODE REVIEW

File: client/src/components/layouts/CollaborationRequest.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  function HandlerRefuse(){
+
+    console.log(`la demande de "${props.username}" a été refusée"`)
+  }
+    return (
+
+ <Container>
+  <Row>
+  <Media>
+  <img
```
Comment (arfa) [inline client/src/components/layouts/CollaborationRequest.tsx]: Utiliser le composant `<Image />`
https://react-bootstrap.github.io/components/images/

Comment (Arfaoui-khouloud) [reply client/src/components/layouts/CollaborationRequest.tsx]: d'accord

Comment (Arfaoui-khouloud) [reply client/src/components/layouts/CollaborationRequest.tsx]: I think it's done

**Review by arfa — APPROVED** [review]
good work

**Review by wassim-arfa — APPROVED** [review]

**Review by layostree — APPROVED** [review]

**Review by Amal-Maalaoui — APPROVED** [review]

**Pull request merged**